### PR TITLE
Adding README.md to examples dir and links to examples github repo in…

### PR DESCRIPTION
**Pull request by lealem47**
… client/server.c

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# wolfSSL examples directory
+#### client and server
+
+ These directories contain a client (client.c) and server (server.c) that utilize a variety of the wolfSSL library's capabilities. The manner in which both programs operate can depend on the configure or can be specified at run-time depending on the end goal. Both applications contain testing as well as benchmarking code.
+
+ ```
+ Compile
```
Comment (dgarske) [inline examples/README.md:7]: Please move the "Compile" out of the triple tick. You can add sh to the triple tick to inform mark-down of format. Same for the other triple tick sections... the title should not be in the shell block.

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+ ```
+
+Run ```./examples/server/server -h``` and ```./examples/client/client -h```  for usage details.
+
+For simpler wolfSSL TLS server/client examples, visit https://github.com/wolfSSL/wolfssl-examples/tree/master/tls
+
+#### echoclient and echoserver
+
+These directories contain a client (echoclient.c) and server (echoserver.c) that establish a connection encrypted by wolfSSL. Like the names indicate, once the connection has been established any messages entered into echoclient are sent to and displayed on the echoserver and are then echoed back to echoclient. The nature of the encryption, as well as additional behavior of the two programs, depends on how wolfSSL was configured ( DTLS enabled/disbled, Filesystem enabled/disabled, etc ... ).
```
Comment (dgarske) [inline examples/README.md:28]: Spelling `disbled`

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# wolfSSL examples directory
+#### client and server
```
Comment (dgarske) [inline examples/README.md:2]: Please use `##` for next level

**Review by dgarske — CHANGES_REQUESTED** [review]
You can preview the mark-down rendering in your branch here:
https://github.com/lealem47/wolfssl/tree/ex-repo-link/examples

Just a couple minor updates.

File: examples/README.md (hunk 2 above)
Comment (lealem47) [reply examples/README.md:28]: Only one typo is a win!

**Review by dgarske — APPROVED** [review]

**Pull request merged**